### PR TITLE
Updated formatting and minor edits for Serverless ApiServerSource docs

### PR DESCRIPTION
**Pull request by abrennan89**
- moved all commands into single command blocks
- updated [source,x] for all code blocks
- updated image version from v0.13.2 to latest

 in serverless ApiServerSource modules

### CODE REVIEW

**Comment by alanfx** [comment]
QE Approved!

File: modules/apiserversource-kn.adoc [hunk 1]
```diff
@@ -13,38 +13,51 @@ This guide describes the steps required to create, manage, and delete an ApiServ
...
 .Procedure
 
-. Create an ApiServerSource event source by entering the following command:
+. Create an ApiServerSource that uses a broker as an event sink, by entering the following command:
 +
+
+[source,terminal]
 ----
-$ kn source apiserver create testevents --sink broker:default --resource "event:v1" --service-account events-sa --mode Resource
+$ kn source apiserver create <event-source-name> --sink broker:<broker-name> --resource "event:v1" --service-account <service-account-name> --mode Resource
```
Comment (lbarbeevargas) [inline modules/apiserversource-kn.adoc:21]: Use underscores and not dashes for any replaceable values in code blocks, for example `event_source_name>` instead of `<event-source-name>`.

File: modules/apiserversource-kn.adoc [hunk 2]
```diff
@@ -13,38 +13,51 @@ This guide describes the steps required to create, manage, and delete an ApiServ
...
 ----
 
-. To check that the ApiServerSource is set up correctly, create a Knative service that dumps incoming messages to its log by entering the following command:
+. Create a Knative service that dumps incoming messages to its log, by entering the following command:
```
Comment (lbarbeevargas) [inline modules/apiserversource-kn.adoc:24]: Same comment as above--remove ", by entering the following command".

File: modules/apiserversource-kn.adoc [hunk 3]
```diff
@@ -13,38 +13,51 @@ This guide describes the steps required to create, manage, and delete an ApiServ
...
 +
+
+[source,terminal]
 ----
-$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+$ kn service create <service-name> --image quay.io/openshift-knative/knative-eventing-sources-event-display:latest
```
Comment (lbarbeevargas) [inline modules/apiserversource-kn.adoc:29]: s/`<service-name>`/`<service_name>`/

File: modules/apiserversource-kn.adoc [hunk 4]
```diff
@@ -13,38 +13,51 @@ This guide describes the steps required to create, manage, and delete an ApiServ
...
 ----
 
-. Create a trigger from the `default` broker to filter events to the service created in the previous step by entering the following `kn` command:
+. Create a trigger to filter events from the `default` broker to the service, by entering the following command:
```
Comment (lbarbeevargas) [inline modules/apiserversource-kn.adoc:32]: Same comment as above--remove ", by entering the following command"

File: modules/apiserversource-kn.adoc [hunk 5]
```diff
@@ -13,38 +13,51 @@ This guide describes the steps required to create, manage, and delete an ApiServ
...
 +
+
+[source,terminal]
 ----
-$ kn trigger create event-display-trigger --sink svc:event-display
+$ kn trigger create <trigger-name> --sink svc:<service-name>
```
Comment (lbarbeevargas) [inline modules/apiserversource-kn.adoc:37]: s/`<trigger-name>`/`<trigger_name>`/
s/`<service-name>`/`<service_name>`/

File: modules/apiserversource-kn.adoc [hunk 6]
```diff
@@ -13,38 +13,51 @@ This guide describes the steps required to create, manage, and delete an ApiServ
...
 ----
 
-. Create events by launching a pod in the default namespace. You can do this by entering the following command:
+. Create events by launching a pod in the default namespace, by entering the following command:
```
Comment (lbarbeevargas) [inline modules/apiserversource-kn.adoc:40]: Same comment as above--remove ", by entering the following command"

File: modules/apiserversource-kn.adoc [hunk 7]
```diff
@@ -13,38 +13,51 @@ This guide describes the steps required to create, manage, and delete an ApiServ
...
 +
+
+[source,terminal]
 ----
 $ oc create deployment hello-node --image=quay.io/openshift-knative/knative-eventing-sources-event-display
 ----
 
-. Check that the controller is mapped correctly by entering the following command and inspecting the output:
+. Check that the controller is mapped correctly, by entering the following command and inspecting the output:
```
Comment (lbarbeevargas) [inline modules/apiserversource-kn.adoc:48]: Deviating some from my previous comments to remove ", by entering the following command:" because it seems important that the user inspects the command output.

s/Check that the controller is mapped correctly, by entering the following command and inspecting the output:/Check that the controller is mapped correctly by inspecting the output to the following command:/

File: modules/apiserversource-kn.adoc [hunk 8]
```diff
@@ -70,15 +83,25 @@ Conditions:
 
 .Verification steps
 
-You can verify that the Kubernetes events were sent into the Knative eventing system by looking at the message dumper function logs.
+You can verify that the Kubernetes events were sent to Knative by looking at the message dumper function logs.
+
+* View the message dumper function logs by entering the following commands:
```
Comment (lbarbeevargas) [inline modules/apiserversource-kn.adoc:88]: I think these two commands should be split up into two steps because you are first getting the pods then retrieving logs for a pod.

File: modules/apiserversource-kn.adoc [hunk 9]
```diff
@@ -113,10 +136,34 @@ Data,
 [id="delete-apiserversource-kn_{context}"]
 == Deleting the ApiServerSource
```
Comment (lbarbeevargas) [inline modules/apiserversource-kn.adoc:137]: Is this a separate procedure from Creating the ApiServerSource? If so, it could be it's own procedure module.

File: modules/apiserversource-kn.adoc [hunk 10]
```diff
@@ -113,10 +136,34 @@ Data,
...
 
-You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding created in this guide by entering the following `kn` and `oc` commands:
```
Comment (lbarbeevargas) [inline modules/apiserversource-kn.adoc:116]: This is a good introduction for the procedure that follows. I would keep it and add a `.Procedure` title afterwards before the steps.

File: modules/apiserversource-kn.adoc [hunk 11]
```diff
@@ -113,10 +136,34 @@ Data,
...
+. Delete the trigger by entering the following command:
```
Comment (lbarbeevargas) [inline modules/apiserversource-kn.adoc:139]: You can remove "by entering the following command" from each of these steps. For example, "Delete the trigger:"

File: modules/apiserversource-yaml.adoc [hunk 12]
```diff
@@ -193,15 +220,25 @@ spec:
 
 .Verification steps
 
-You can verify that the Kubernetes events were sent into the Knative eventing system by looking at the message dumper function logs.
+You can verify that the Kubernetes events were sent to Knative by looking at the message dumper function logs.
+
+* View the message dumper function logs by entering the following commands:
```
Comment (lbarbeevargas) [inline modules/apiserversource-yaml.adoc:225]: Same as the previous comment, each command should have its own step.

File: modules/apiserversource-yaml.adoc [hunk 13]
```diff
@@ -236,10 +273,34 @@ Data,
 [id="delete-apiserversource-yaml_{context}"]
```
Comment (lbarbeevargas) [inline modules/apiserversource-yaml.adoc:273]: Same comment as for previous module--should Deleting the ApiServerSource be its own procedure module?

File: modules/apiserversource-yaml.adoc [hunk 14]
```diff
@@ -236,10 +273,34 @@ Data,
...
 == Deleting the ApiServerSource
 
-You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding created in this guide by entering the following `oc` commands:
```
Comment (lbarbeevargas) [inline modules/apiserversource-yaml.adoc:239]: This is a good introduction for the procedure that follows. I would keep it and add a .Procedure title afterwards before the steps.

File: modules/apiserversource-yaml.adoc [hunk 15]
```diff
@@ -236,10 +273,34 @@ Data,
...
+. Delete the trigger by entering the following command:
```
Comment (lbarbeevargas) [inline modules/apiserversource-yaml.adoc:276]: You can remove "by entering the following command" from each of these steps. For example, "Delete the trigger:"

File: modules/apiserversource-kn.adoc [hunk 16]
```diff
@@ -113,10 +136,34 @@ Data,
...
+
+. Delete the event source by entering the following command:
++
+
+[source,terminal]
+----
+$ kn source apiserver delete <source-name>
+----
+
+. Delete the service account, cluster role, and cluster binding created in this guide by entering the following command:
```
Comment (lbarbeevargas) [inline modules/apiserversource-kn.adoc:163]: s/and cluster binding created in this guide by entering the following command:/and cluster binding:/

File: modules/apiserversource-yaml.adoc [hunk 17]
```diff
@@ -236,10 +273,34 @@ Data,
...
+
+. Delete the event source by entering the following command:
++
+
+[source,terminal]
+----
+$ oc delete -f k8s-events.yaml
+----
+
+. Delete the service account, cluster role, and cluster binding created in this guide by entering the following command:
```
Comment (lbarbeevargas) [inline modules/apiserversource-yaml.adoc:300]: s/and cluster binding created in this guide by entering the following command:/and cluster binding:/

**Review by lbarbeevargas — COMMENTED** [review]
Hey @abrennan89, great job! Just a few comments and a suggestion to see if the "Deleting the ApiServerSource" sections need to be split into their own procedure modules. 

@bobfuru is going to take a look through this PR as well in case I missed anything or my comments need to be corrected.

File: modules/apiserversource-yaml.adoc [hunk 18]
```diff
@@ -193,15 +220,25 @@ spec:
...
+
+* View the message dumper function logs by entering the following commands:
 
-You can view the message dumper function logs by entering the following commands:
+[source,terminal]
 ----
 $ oc get pods
+----
+
++
```
Comment (lbarbeevargas) [inline modules/apiserversource-yaml.adoc:232]: This `+` renders in the output. It can be removed when the two commands are separated into their own steps.

File: modules/apiserversource-kn.adoc (hunk 16 above)
Comment (bobfuru) [inline modules/apiserversource-kn.adoc:163]: Agree with this and suggest removing "this guide" occurrences throughout and replacing with "this section" if necessary.

**Review by bobfuru — COMMENTED** [review]
Agree with Lindsey's comments and added one more suggestion to remove or replace "this guide" when it doesn't occur at the assembly-level in the beginning.

**Comment by abrennan89** [comment]
@lbarbeevargas @bobfuru thanks so much for the great reviews folks!
Feedback should all be addressed now if you can PTAL :slightly_smiling_face: 

Once this is approved it can be merged I think, since it's not waiting for a particular Serverless release. It needs to be CP'd to OCP 4.3, 4.4, 4.5 and 4.6 docs, so lmk if you'd like me to open PRs to do this or if one of you can do this when merging :slightly_smiling_face: thanks again.

File: modules/apiserversource-kn-delete.adoc [hunk 19]
```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-kn-source.adoc
+
+[id="delete-apiserversource-kn_{context}"]
+= Deleting the ApiServerSource using the Knative CLI (`kn`)
+
+You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding created in this guide by entering the following `kn` and `oc` commands.
```
Comment (bobfuru) [inline modules/apiserversource-kn-delete.adoc:8]: I'd reword this slightly:

> You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding using `kn` and `oc` commands.

Or better yet, match the wording used in the next section:

> This section describes the steps used to delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding using `kn` and `oc` commands.

File: modules/apiserversource-kn.adoc [hunk 20]
```diff
@@ -5,46 +5,59 @@
...
+. Create events by launching a pod in the default namespace:
 +
+
+[source,terminal]
 ----
 $ oc create deployment hello-node --image=quay.io/openshift-knative/knative-eventing-sources-event-display
 ----
 
-. Check that the controller is mapped correctly by entering the following command and inspecting the output:
+. Check that the controller is mapped correctly by inspecting the output to the following command:
```
Comment (bobfuru) [inline modules/apiserversource-kn.adoc:48]: s/output to/output generated by/ ?

File: modules/apiserversource-kn.adoc [hunk 21]
```diff
@@ -70,15 +83,29 @@ Conditions:
 
 .Verification steps
 
-You can verify that the Kubernetes events were sent into the Knative eventing system by looking at the message dumper function logs.
+You can verify that the Kubernetes events were sent to Knative by looking at the message dumper function logs.
+
+. Get the pods:
```
Comment (bobfuru) [inline modules/apiserversource-kn.adoc:88]: s/pods/Pods/

File: modules/apiserversource-kn.adoc [hunk 22]
```diff
@@ -70,15 +83,29 @@ Conditions:
...
++
 
-You can view the message dumper function logs by entering the following commands:
+[source,terminal]
 ----
 $ oc get pods
+----
+
+. View the message dumper function logs for the pods:
```
Comment (bobfuru) [inline modules/apiserversource-kn.adoc:96]: s/pods/Pods/

File: modules/apiserversource-yaml-delete.adoc [hunk 23]
```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-kn-source.adoc
+
+[id="delete-apiserversource-yaml_{context}"]
+= Deleting the ApiServerSource
+
+You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding created in this guide by deleting the following YAML files.
```
Comment (bobfuru) [inline modules/apiserversource-yaml-delete.adoc:8]: Similar comment to above, for consistency:

> This section describes how to delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding deleting their YAML files.

Your call. :)

File: modules/apiserversource-yaml.adoc [hunk 24]
```diff
@@ -14,10 +14,10 @@ This guide describes the steps required to create, manage, and delete an ApiServ
 
 .Procedure
 
-. Create a service account, role, and role binding for the ApiServerSource.
-+
-You can do this by creating a file named `authentication.yaml` and copying the following sample code into it:
+. Create a service account, role, and role binding for the ApiServerSource by creating a file named `authentication.yaml` and copying the following sample code into it:
```
Comment (bobfuru) [inline modules/apiserversource-yaml.adoc:17]: Nitpick, but it might read better with more active voice, such as:

> To create a service account, role, and role binding for the ApiServerSource,  create a file named `authentication.yaml` and copy the following sample code into it:

File: modules/apiserversource-yaml.adoc [hunk 25]
```diff
@@ -64,16 +65,18 @@ subjects:
...
+[source,terminal]
 ----
-$ oc apply --filename authentication.yaml
+$ oc apply -f authentication.yaml
 ----
 
-. Create an ApiServerSource event source.
-+
-You can do this by creating a file named `k8s-events.yaml` and copying the following sample code into it:
+. Create an ApiServerSource event source by creating a file named `k8s-events.yaml` and copying the following sample code into it:
```
Comment (bobfuru) [inline modules/apiserversource-yaml.adoc:76]: same comment about active voice:

> To create an ApiServerSource event source, create a file named `k8s-events.yaml` and copy the following sample code into it:

File: modules/apiserversource-yaml.adoc [hunk 26]
```diff
@@ -64,16 +65,18 @@ subjects:
...
 ====
 +
-After you have created the `authentication.yaml` file, apply it by entering the following command:
+After you have created the `authentication.yaml` file, apply it:
 +
+
+[source,terminal]
 ----
-$ oc apply --filename authentication.yaml
+$ oc apply -f authentication.yaml
```
Comment (bobfuru) [inline modules/apiserversource-yaml.adoc:73]: Should these files have 3-letter extensions (`.yml`) or does that even matter?

File: modules/apiserversource-yaml.adoc [hunk 27]
```diff
@@ -114,17 +122,20 @@ spec:
...
+[source,terminal]
 ----
-$ oc apply --filename service.yaml
+$ oc apply -f service.yaml
 ----
 
-. Create a trigger from the `default` broker to filter events to the service created in the previous step.
-+
-You can create the trigger by creating a file named `trigger.yaml` and copying the following sample code into it:
+. Create a trigger from the `default` broker to filter events to the service created in the previous step by creating a file named `trigger.yaml` and copying the following sample code into it:
```
Comment (bobfuru) [inline modules/apiserversource-yaml.adoc:135]: same comment about active voice ("To create a trigger...create a file named...")

File: modules/apiserversource-yaml.adoc [hunk 28]
```diff
@@ -138,27 +149,37 @@ spec:
...
 +
+
+[source,terminal]
 ----
-$ oc apply --filename trigger.yaml
+$ oc apply -f trigger.yaml
 ----
 
-. Create events by launching a pod in the default namespace. You can do this by entering the following command:
+. Create events by launching a pod in the default namespace:
```
Comment (bobfuru) [inline modules/apiserversource-yaml.adoc:162]: > To create events, launch a Pod in the default namespace:

File: modules/apiserversource-yaml.adoc [hunk 29]
```diff
@@ -193,15 +214,29 @@ spec:
 
 .Verification steps
 
-You can verify that the Kubernetes events were sent into the Knative eventing system by looking at the message dumper function logs.
+You can verify that the Kubernetes events were sent to Knative by looking at the message dumper function logs.
```
Comment (bobfuru) [inline modules/apiserversource-yaml.adoc:217]: "To verify that..."

**Review by bobfuru — COMMENTED** [review]
Some more personal opinion-type choices for your consideration. 😁

File: modules/apiserversource-yaml.adoc (hunk 26 above)
Comment (abrennan89) [reply modules/apiserversource-yaml.adoc:73]: I think either one is fine, I'll leave as .yaml for now for consistency since there are a ton of them, I don't want to change just one.

File: modules/apiserversource-yaml.adoc [hunk 30]
```diff
@@ -114,17 +122,20 @@ spec:
...
+[source,terminal]
 ----
-$ oc apply --filename service.yaml
+$ oc apply -f service.yaml
 ----
 
-. Create a trigger from the `default` broker to filter events to the service created in the previous step.
-+
-You can create the trigger by creating a file named `trigger.yaml` and copying the following sample code into it:
+. To create a trigger from the `default` broker, that filters events to the service created in the previous step, create a file named `trigger.yaml` and copying the following sample code into it:
```
Comment (bobfuru) [inline modules/apiserversource-yaml.adoc:135]: s/broker, that/broker that/
s/and copying/and copy/

File: modules/apiserversource-yaml.adoc [hunk 31]
```diff
@@ -193,15 +214,29 @@ spec:
 
 .Verification steps
 
-You can verify that the Kubernetes events were sent into the Knative eventing system by looking at the message dumper function logs.
+To verify that the Kubernetes events were sent to Knative you can look at the message dumper function logs.
+
+. Get the pods:
```
Comment (bobfuru) [inline modules/apiserversource-yaml.adoc:219]: s/pods/Pods

File: modules/apiserversource-yaml.adoc [hunk 32]
```diff
@@ -193,15 +214,29 @@ spec:
...
++
 
-You can view the message dumper function logs by entering the following commands:
+[source,terminal]
 ----
 $ oc get pods
+----
+
+. View the message dumper function logs for the pods:
```
Comment (bobfuru) [inline modules/apiserversource-yaml.adoc:227]: s/pods/Pods/

File: modules/apiserversource-kn.adoc [hunk 33]
```diff
@@ -5,46 +5,59 @@
...
 +
+
+[source,terminal]
 ----
-$ kn trigger create event-display-trigger --sink svc:event-display
+$ kn trigger create <trigger_name> --sink svc:<service_name>
 ----
 
-. Create events by launching a pod in the default namespace. You can do this by entering the following command:
+. Create events by launching a pod in the default namespace:
```
Comment (bobfuru) [inline modules/apiserversource-kn.adoc:40]: s/pod/Pod/

File: modules/apiserversource-yaml.adoc [hunk 34]
```diff
@@ -138,27 +149,37 @@ spec:
...
+. To create events, launch a Pod in the default namespace:
 +
+
+[source,terminal]
 ----
 $ oc create deployment hello-node --image=quay.io/openshift-knative/knative-eventing-sources-event-display
 ----
 
-. Check that the controller is mapped correctly by entering the following command and inspecting the output:
+. To check that the controller is mapped correctly, enter the following command and inspecting the output:
```
Comment (bobfuru) [inline modules/apiserversource-yaml.adoc:170]: s/and inspecting/and inspect/

File: modules/apiserversource-yaml.adoc [hunk 35]
```diff
@@ -193,15 +214,29 @@ spec:
 
 .Verification steps
 
-You can verify that the Kubernetes events were sent into the Knative eventing system by looking at the message dumper function logs.
+To verify that the Kubernetes events were sent to Knative you can look at the message dumper function logs.
```
Comment (bobfuru) [inline modules/apiserversource-yaml.adoc:217]: s/Knative you/Knative, you/

**Review by bobfuru — COMMENTED** [review]
_almost there_ 😁  - just a few more small nits, thanks!
